knowledge_repo/app/config_defaults.py

```diff
@@ -171,6 +171,18 @@ def AUTH_USER_ATTRIBUTE_SETTER(user):
 # https://flask-login.readthedocs.io/en/latest/#cookie-settings
 REMEMBER_COOKIE_DURATION = datetime.timedelta(days=365)
 
+# ---------------------------------------------------
+# LDAP configuration
+# ---------------------------------------------------
+# When using an LDAP server for user verification, you need to configure
+# the location of the server, and the directory structure used by your
+# organization.
+
+# Currently the port and protocol must both be included in the server address
+LDAP_SERVER = 'ldaps://127.0.0.1:389'
```

Comment:
Let's comment it out by default so that if not present an appropriate error message is thrown by the LDAP AuthProvider.